submissions/team_api.py

```diff
@@ -34,7 +34,8 @@ def create_submission_for_team(
     """
     This api function:
       1. Creates a `TeamSubmission` record, and
-      2. Creates `Submission` records for every member of the team by calling api.create_submission()
+      2. Creates `Submission` records for ONLY those team members that don't have any past submissions
```

Comment:
Nit: I'd leave out the comment about it calling `api.create_submission()`, seems like an unnecessary detail which ties high-level docstring too much to implementation detail.